lms/views/admin/application_instance.py

```diff
@@ -252,37 +252,54 @@ def show_instance(self):
         return {"instance": ai}
 
     @view_config(
-        route_name="admin.instance.id",
-        request_method="POST",
-        require_csrf=True,
-    )
-    @view_config(
-        route_name="admin.instance.consumer_key",
+        route_name="admin.instance.move_org",
         request_method="POST",
         require_csrf=True,
     )
-    def update_instance(self):
+    def move_application_instance_org(self):
         ai = self._get_ai_or_404(**self.request.matchdict)
 
         try:
             self.application_instance_service.update_application_instance(
                 ai,
-                lms_url=self.request.params.get("lms_url", "").strip(),
-                deployment_id=self.request.params.get("deployment_id", "").strip(),
-                developer_key=self.request.params.get("developer_key", "").strip(),
-                developer_secret=self.request.params.get(
-                    "developer_secret", ""
-                ).strip(),
                 organization_public_id=self.request.params.get(
                     "org_public_id", ""
                 ).strip(),
             )
+            self.request.session.flash(
+                f"Updated application instance {ai.id}", "messages"
+            )
         except ValidationError as err:
             self.request.session.flash(err.messages, "validation")
             return HTTPFound(
                 location=self.request.route_url("admin.instance.id", id_=ai.id)
             )
 
+        return HTTPFound(
```

Comment:
This view is a good candidate to have the audit trail call. Not in this PR as it has some extra complexity.